ansible-ipi-install/roles/installer/tasks/40_create_manifest.yml

```diff
@@ -16,14 +16,3 @@
   - "{{ dir }}/openshift"
   - "{{ dir }}/manifests"
   tags: manifests
-
-- name: Copy the metal3-config.yaml to {{ dir }}/openshift directory
-  copy:
-    src: "{{ dir }}/metal3-config.yaml"
-    dest: "{{ dir }}/openshift/98_metal3-config.yaml"
-    owner: "{{ ansible_user }}"
-    group: "{{ ansible_user }}"
-    mode: '0644'
-    remote_src: yes
-  when: (release_version[0]|int == 4) and (release_version[2]|int <= 3)
-  tags: manifests
```

Comment:
we needed this for versions 4.3 and lower. Does it matter to leave in?

Reply:
openshift 4.3 is no longer supported (since Oct 2020), at the very least we'd need to change the version check to 
> (release_version is version("4.3", "<="))

but it seems like we'd be keeping it around for no reason?